ci-operator/config/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master.yaml

```diff
@@ -51,38 +51,6 @@ tests:
   commands: make unit
   container:
     from: src
```

Comment:
I need to double-check whether this already lives as a GH action in the upstream OLM repository.

Reply:
It does

Reply:
Including that unit test job, here are the only remaining jobs in the master/4.8+ branch prow configurations:

```
- as: verify
  commands: make verify
  container:
    from: src
- as: unit
  commands: make unit
  container:
    from: src
```

Let's get this in first, and I can look into ensuring those jobs live as GH actions before removing the master/release-4.8+ prow configuration entirely for upstream OLM.